build_containers_and_push.sh

```diff
@@ -7,7 +7,7 @@ function echo_run() {
     "$@"
 }
 
-for spackver in "v0.18.1" "v0.19.2" ; do
+for spackver in "v0.19.2" "v0.20.1" ; do
```

Comment:
Last request for change: v0.20.2 was released recently, so just upgrade the version here too.